spring-boot-project/spring-boot-autoconfigure/src/test/resources/hazelcast.xml

```diff
@@ -5,7 +5,6 @@
 	<map name="defaultCache" />
 	<network>
 		<join>
-			<tcp-ip enabled="false" />
```

Comment:
This seems an unrelated change. I am happy to polish the support in a separate commit though. Can you explain why you've removed this line?

Reply:
Yeah, it's just not used. Reverted. Can be sent in a separate PR.